indico/modules/events/timetable/client/js/utils.ts

```diff
@@ -100,6 +101,7 @@ export const mapTTDataToEntry = (data): TopLevelEntry => {
     textColor: colors ? colors.text : '',
     backgroundColor: colors ? colors.background : '',
     sessionId: sessionId || null,
+    parentId: parentId || null,
```

Comment:
The `parentId` should be absent for top-level entries. You could do something like this:
```js
const mappedObj = {...}
if (parentId) {
  mappedObj.parentId = parentId
}
```

Reply:
You mean removing the `parentId: parentId || null,` line and adding these other ones? Why is it better to have it absent rather than having it set to `null`?

Reply:
Makes sense!